package.json

```diff
@@ -40,6 +40,10 @@
     "eslint-plugin-prettier": "^2.3.1",
     "extract-text-webpack-plugin": "^3.0.1",
     "ff": "0.2.2",
+    "fluent": "^0.4.2",
```

Comment:
I published `fluent` 0.6.0 last night. It has support for Fluent Syntax 0.5 which will soon be the only one used by Pontoon. It also supports Fluent Syntax 0.4.